modules/apps/portal-cache/portal-cache-ehcache-impl/src/main/resources/ehcache/liferay-multi-vm.xml

```diff
@@ -8,6 +8,8 @@
 	Caches in this file can be clustered and are configured to be clustered.
 	-->
 
+	<persistence directory="${liferay.home}/data/ehcache"/>
```

Comment:
@dantewang We do not have this line in ehacahe 2 xml before, why you say it is for consistency with ehcache 2